kong/key.go

```diff
@@ -0,0 +1,22 @@
+package kong
```

Comment:
There's maybe not a particularly good reason for it, but types that are managed through a go-kong service are currently all collected in types.go, so I'd say move these two inside there as well to keep in consistent. If we want to split those out somehow we should do it all in one go.

Reply:
moved!